README.md

```diff
@@ -101,44 +101,46 @@ MIT Licensed. See [LICENSE](https://github.com/terraform-aws-modules/terraform-a
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
-| cluster_create_timeout | Timeout value when creating the EKS cluster. | string | `15m` | no |
-| cluster_delete_timeout | Timeout value when deleting the EKS cluster. | string | `15m` | no |
-| cluster_name | Name of the EKS cluster. Also used as a prefix in names of related resources. | string | - | yes |
-| cluster_security_group_id | If provided, the EKS cluster will be attached to this security group. If not given, a security group will be created with necessary ingres/egress to work with the workers and provide API access to your current IP/32. | string | `` | no |
-| cluster_version | Kubernetes version to use for the EKS cluster. | string | `1.10` | no |
-| config_output_path | Where to save the Kubectl config file (if `write_kubeconfig = true`). Should end in a forward slash `/` . | string | `./` | no |
-| kubeconfig_aws_authenticator_additional_args | Any additional arguments to pass to the authenticator such as the role to assume. e.g. ["-r", "MyEksRole"]. | list | `<list>` | no |
-| kubeconfig_aws_authenticator_command | Command to use to to fetch AWS EKS credentials. | string | `aws-iam-authenticator` | no |
-| kubeconfig_aws_authenticator_env_variables | Environment variables that should be used when executing the authenticator. e.g. { AWS_PROFILE = "eks"}. | map | `<map>` | no |
-| kubeconfig_name | Override the default name used for items kubeconfig. | string | `` | no |
-| manage_aws_auth | Whether to write and apply the aws-auth configmap file. | string | `true` | no |
-| map_accounts | Additional AWS account numbers to add to the aws-auth configmap. See examples/eks_test_fixture/variables.tf for example format. | list | `<list>` | no |
-| map_roles | Additional IAM roles to add to the aws-auth configmap. See examples/eks_test_fixture/variables.tf for example format. | list | `<list>` | no |
-| map_users | Additional IAM users to add to the aws-auth configmap. See examples/eks_test_fixture/variables.tf for example format. | list | `<list>` | no |
+| cluster\_create\_timeout | Timeout value when creating the EKS cluster. | string | `15m` | no |
```

Comment:
Where do all these extra slashes come from? Did your IDE or editor add these automatically?

Reply:
`terraform-docs` added them.

Reply:
Ahh it's from this PR: https://github.com/segmentio/terraform-docs/pull/63
Kinda annoying.

Reply:
@max-rocket-internet It is a one-time operation to reformat markdown files. I am working on applying this fix in all repositories under terraform-aws-modules now.

Reply:
OK cool. I get the PR but I've never seen any italic in variable names in markdown tables on Github from this issue. Perhaps it's a problem for strict markdown as opposed to github markdown?

Reply:
AFAIK you are right, but I didn't investigate deep. I trust @metmajer on making a good job with `terraform-docs` :)

Reply:
Feeling honored :) Guys, have you considered using the `with-aggregate-type-defaults` option with terraform-docs? This would allow you to add a variable's default value into the documentation (as opposed to just `<list>` or `<map>`).

Reply:
@max-rocket-internet I've just released a new version of https://github.com/antonbabenko/pre-commit-terraform where this is implemented if you want to give it a try. Kudos to @metmajer :)

Reply:
@metmajer 

> have you considered using the with-aggregate-type-defaults option with terraform-docs?

I just tried this:

```
terraform-docs md --with-aggregate-type-defaults ./ | cat -s | tail -r | tail -n +2 | tail -r > README.md
```

But it didn't make any changes. Is that expected? Or does that mean we had all our inputs and outputs already perfectly labeled with correct defaults?

Reply:
Unfortunately, the opts parser is strict on where the long-opt is provided. Please give the following command a try:

```
terraform-docs --with-aggregate-type-defaults md ./
```

Reply:
Ahh I get you. Love it! I'll make a PR.